### DISCUSSION TITLE
Add a `!pip install instructor` cell to avoid Pydantic V1 issue in colab

### DISCUSSION THREAD

**Discussion by sandeshrajbhandari** [567-labs/instructor#204] [General]
**Is your feature request related to a problem? Please describe.**
Colab has Pydantic V1 installed, so it gives following no attribute error. 
```
AttributeError                            Traceback (most recent call last)
<ipython-input-19-d26671d71c0f> in <cell line: 2>()
      1 # Data is validated to get better error messages
----> 2 person = Person.model_validate({"name": "Sam", "age": "30.2"})
      3 person

AttributeError: type object 'Person' has no attribute 'model_validate'
```

SOLUTION:
either add a `!pip install -U pydantic` cell to use the latest pydantic to avoid this error, or just install instructor to install required dependencies including new pydantic.

**Comment by tinycrops** [comment]
Hi! Welcome to the community.  What your describing is best resolved by a pull request, in my opinion.  Here's a quick run down on how to make that happen https://chat.openai.com/share/dd75d8ac-30ad-4e43-afa9-4dc6d595e92e
Also, I found as the excitement of watching my computer learn a new library wore off, it's best to add 'q' to pip installs:
```
!pip install -Uq pydantic
```
